ui/apps/platform/src/Containers/VulnMgmt/Entity/Component/VulnMgmtComponentOverview.js

```diff
@@ -73,9 +75,11 @@ function VulnMgmtComponentOverview({ data, entityContext }) {
         );
     }
 
-    const currentEntity = { [entityTypes.COMPONENT]: id };
+    const currentEntity = { [currentEntityType]: id };
     const newEntityContext = { ...entityContext, ...currentEntity };
 
+    const entityGridContainerClassName = `${entityGridContainerBaseClassName} grid-columns-1 md:grid-columns-2 lg:grid-columns-2`;
```

Comment:
So this is the before and after of this change

Before:
<img width="1552" alt="Screen Shot 2022-06-29 at 10 59 12 PM" src="https://user-images.githubusercontent.com/4805485/176603596-711351c5-76e0-4485-b711-d504b6be6c2e.png">

After:
<img width="1552" alt="Screen Shot 2022-06-29 at 10 59 15 PM" src="https://user-images.githubusercontent.com/4805485/176603620-ceb8ae3e-4fae-490a-ad6d-d771adab0d06.png">

I noticed that we were only showing two columns in the component overview so it differed from the class we were using in general, which was for 3 columns. There is a base class which I use and then append classes specific for this view

Reply:
^ That's probably a side-effect of widening the tiles right sidebar.

I'm fine with your fix for it, but I'm afraid this may turn into whack-a-mole. Let's get UX suggestion on this. cc: @alanonthegit

Reply:
Oh no, that wasn't the issue. We're using grid columns for the overview and it was using a 3 column grid before. If you look at the first image you'll see it looks like you can add another third element to the right. It was on master as well. It's not really related to splitting the components but I just did it anyway.